docs/reference/execute-methods.md

```diff
@@ -2086,3 +2086,24 @@ Same as for [mobile: startXCTestScreenRecording](#mobile-startxctestscreenrecord
 Name | Type | Description | Example
 --- | --- | --- | ---
 payload | string | Base64-encoded content of the recorded media file if `remotePath` parameter is empty/null or an empty string otherwise. The resulting media is expected to a be a valid QuickTime movie (.mov). | `YXBwaXVt....`
+
+### mobile: simctl
+
+Runs the given command as a subcommand of `xcrun simctl`.
+This method requires `simctl` [server secuity option](https://appium.io/docs/en/latest/guides/security/).
+
+`stream` commands such as log stream keeps listening the output, thus Appium keeps waiting for the command to end.
```

Comment:
have you tested it? Under the hood teen_process's exec is used, which is always limited by a timeout (20 seconds by default)

Reply:
in general it is always better/safer to cap the maximum duration

Reply:
This is by https://github.com/appium/node-simctl/blob/9ff3b4eb12c20ba29f3e6918577762ec1bcde26d/lib/simctl.js#L170, so the default timeout is 10mins. I'll add timeout option to the lib so that we can give `timeout` option to the `exec` and argument